### PR TITLE
Point meetings doc to YouTube and organize by committee

### PR DESCRIPTION
**Pull request by scottrigby**
Resolves https://github.com/gitops-working-group/gitops-working-group/issues/71 and https://github.com/gitops-working-group/gitops-working-group/discussions/123.

I've already re-organized the meeting Google doc by committee, to make meetings easier to read, find, and know where to add the next one by browsing the table of contents.

### CODE REVIEW

File: MEETINGS.md [hunk 1]
```diff
@@ -1,32 +1,27 @@
...
-- [Meeting Notes](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/)
+See the [Google Doc](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/) for detailed join info, agenda, and notes for all GitOps Working Group meetings.
 
-### Thursday, February 11th at 18:00 GMT
+## Working Group Monthly
 
-- [Video Recording](https://weaveworks.zoom.us/rec/share/OPy5MSMzgIkPX1-PAdhwrnGGa82Lr5BuVpQ1iSK-sbHPph68ML1JNVQXRKlP09bw.bjqDnPKEA2W9D2HG)
-- Passcode: `fwPy8^fb`
-- [Meeting Notes](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/)
+- May 13, 2021: [join link and agenda](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit#heading=h.lq3s56hyydm0) ⚠️ Note: May Monthly meeting moved to 3rd Thursday due to KubeCon EU
```
Comment (lloydchang) [inline MEETINGS.md:11]:
```Suggestion
- May 13, 2021: [join link and agenda](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit#heading=h.lq3s56hyydm0)
```
@scottrigby May 13th is the second Thursday, whereas KubeCon EU is during the first Thursday according to https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/

Comment (scottrigby) [reply MEETINGS.md:11]: You are correct 😅

**Comment by lloydchang** [comment]
Zoom recordings have Chat Messages that aren't in YouTube.  Unfortunately, I don't see an option to upload Chat Messages to YouTube based on https://support.google.com/youtube/answer/57407

Participants rely on meeting moderators to read Chat Messages aloud to relay their opinions to the audience.

For example:
• Are there Chat Messages for [GitOps Principles Committee - April 14, 2021](https://www.youtube.com/watch?v=zWxAZuHNGYM) from participants who didn't raise their hands to speak?
• "+1" can appear in Chat Messages.
• When you time permits, can Zoom recording for [GitOps Principles Committee - April 14, 2021](https://www.youtube.com/watch?v=zWxAZuHNGYM) be added to [Google Doc: April 14, 2021]( https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit#heading=h.7wmerton06if)?

Thanks @scottrigby

File: MEETINGS.md [hunk 2]
```diff
@@ -1,32 +1,27 @@
...
+- January 14, 2021: [recording](https://www.youtube.com/watch?v=JypiRn8HTbw&list=PLXOML2VBdIo4-biBmCbfNkP0ywF0f5mau&index=2)
+- December 10, 2020: [recording](https://www.youtube.com/watch?v=LnzIE6tDfbQ&list=PLXOML2VBdIo4-biBmCbfNkP0ywF0f5mau&index=1)
 
-### Thursday, January 14th at 17:00 GMT
+## GitOps Principles Committee Meetings
 
-- [Video Recording](https://weaveworks.zoom.us/rec/share/_qrMRze16IHb2d4KIOHSa2Wxb_XzbU41-ZH6YZI5aJkUqOHM1bBQLCQhkFva5xo0.ay0LdxTVPCTlgbSd)<br>
-- Passcode: `EpQ26^cV`
-- [Meeting Notes](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/)
+- April 14, 2021: [recording](https://www.youtube.com/watch?v=zWxAZuHNGYM&list=PLXOML2VBdIo6XfUTaIbanBN2fIDEyR25s&index=3)
```
Comment (lloydchang) [inline MEETINGS.md:20]: For example:
• Are there Chat Messages for [GitOps Principles Committee - April 14, 2021](https://www.youtube.com/watch?v=zWxAZuHNGYM) from participants who didn't raise their hands to speak?
• "+1" can appear in Chat Messages.
• When you time permits, can Zoom recording for [GitOps Principles Committee - April 14, 2021](https://www.youtube.com/watch?v=zWxAZuHNGYM) be added to [Google Doc: April 14, 2021]( https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit#heading=h.7wmerton06if)?

Thanks @scottrigby

**Review by lloydchang — APPROVED** [review]

File: MEETINGS.md (hunk 2 above)
Comment (scottrigby) [reply MEETINGS.md:20]: Yes that is annoying. The goal here was to make the meetings more easily accessible, in a consistent format. I agree the zoom text chat is an important part of the conversation, and should not be left out.

I was able to download all the zoom chat text files separately, and could link to each in the YouTube video description. We could host the text files themselves in this repo, in a `meetings` directory. A slightly more organized version of https://github.com/gitops-working-group/gitops-working-group/issues/9. I'll update this PR

Comment (scottrigby) [reply MEETINGS.md]: OK added text chat files, and linked from MEETINGS.md. Once merged, we can also link to these from the YouTube video description for each.

Comment (lloydchang) [reply MEETINGS.md]: From [Google Doc: December 10, 2020](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit#heading=h.e0ignly90e17):
> [Omer] Chat can be saved in new versions of zoom, Scott will save

-----

@mjpitz [asked:](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit?disco=AAAAHdlWQWw)
> Is this available anywhere?

Yes.  It's available at:
https://github.com/scottrigby/gitops-working-group/blob/meetings-youtube/meetings/monthly/2020-12-10-chat.txt
and
https://weaveworks.zoom.us/rec/share/jyagKQsL7irvi1pF-RHzrrwH-Sqa59wNnBK0F2H8QYBo_rxJZOXMO-5j5CwUgKiK.9cl0var5BN-z0qrI
Passcode: `%mi$&f8e`

@mjpitz Thanks for asking about meeting chats.

@scottrigby Thanks for publishing them.

**Review by lloydchang — APPROVED** [review]

**Review by lloydchang — APPROVED** [review]
Approved https://github.com/gitops-working-group/gitops-working-group/pull/133.

Thanks @chrisk700 @scottrigby

**Review by dj1k — APPROVED** [review]
Looks good to me.

**Comment by todaywasawesome** [comment]
Looks good merging.

**Pull request merged**